packages/react-provider/src/types.ts

```diff
@@ -32,4 +33,5 @@ export interface ICourierContext {
   userSignature?: string;
   wsUrl?: string;
   middleware?: any;
+  onMessage?: Interceptor;
```

Comment:
`onMessage` interface requires a return value, is that useful or can it be `void`? we don't seem to be using the return value from onMessage?

Reply:
we are, i pass onMessage in as `courier.intercept(onMessage)` and if onMessage returns without a message, it will prevent a message from showing